### PR TITLE
Add backupStats func to docs

### PR DESCRIPTION
**Pull request by DeepikaDixit**
## Change Overview

Add `backupStats` func to docs

## Pull request type

Please check the type of change your PR introduces:
- [X] Work in Progress
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Trivial/Minor
- [ ] Bugfix
- [ ] Feature
- [X] Documentation

## Issues

- #XXX

## Test Plan

- [ ] Manual
- [ ] Unit test
- [ ] E2E

### CODE REVIEW

File: docs/functions.rst [hunk 1]
```diff
@@ -829,6 +829,63 @@ Example:
...
+  :linenos:
+
+  actions:
+    backupStats:
+      type: Deployment
+      outputArtifacts:
+        backupStats:
+          keyValue:
+            mode: "{{ .Phases.backupDataStatsFromObjectStore.Output.BackupDataStatsOutputMode }}"
+            fileCount "{{ .Phases.backupDataStatsFromObjectStore.Output.BackupDataStatsOutputFileCount }}"
```
Comment (tdmanv) [inline docs/functions.rst]:
```suggestion
            fileCount: "{{ .Phases.backupDataStatsFromObjectStore.Output.BackupDataStatsOutputFileCount }}"
```

File: docs/functions.rst [hunk 2]
```diff
@@ -829,6 +829,63 @@ Example:
...
+            size: "{{ .Phases.backupDataStatsFromObjectStore.Output.BackupDataStatsOutputSize }}"
+      phases:
+        - func: BackupData
+          name: BackupToObjectStore
+          args:
+            namespace: "{{ .Deployment.Namespace }}"
+            backupArtifactPrefix: s3-bucket/path/artifactPrefix
+            mode: restore-size
+            backupID: "{{ .ArtifactsIn.snapshot.KeyValue.backupIdentifier }}"
+
```
Comment (tdmanv) [inline docs/functions.rst]:
```suggestion
```

**Comment by DeepikaDixit** [comment]
@tdmanv mergify beat me to it! I’ll send an update with your changes

**Comment by tdmanv** [comment]
doh, my bad - I should have added DNM

**Pull request merged**